packages/dnb-eufemia/src/components/icon/Icon.tsx

```diff
@@ -47,7 +48,7 @@ type IconType =
   | false
 
 /** For external usage */
-export type IconIcon = IconType | React.FC<unknown>
+export type IconIcon = IconType | FormStatusIconTypes | React.FC<any>
```

Comment:
```suggestion
export type IconIcon = IconType | FormStatusIconTypes
```

Or should I do this?

Reply:
I think having `FormStatusIconTypes` like you have done, is good. 

Changing the unknown to `React.FC<any>` would have solved the "issue" by its own I guess.

But can we leave it `React.FC<unknown>` as before, and it would solve the issue by adding `FormStatusIconTypes`?